Other/AninaQuestRewards.cs

```diff
@@ -0,0 +1,145 @@
+/*
+name: AninaQuestRewards
+description: This Script will farm all the items in /lavarockshore from the NPC Anina
+tags: forge, blaze, awe, blazelight, molten, core, burning, doom
+*/
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/Good/GearOfAwe/BladeOfAwe.cs
+//cs_include Scripts/Farm/BuyScrolls.cs
+//cs_include Scripts/CoreDailies.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/Good/BLoD/CoreBLOD.cs
+//cs_include Scripts/Evil/NSoD/CoreNSOD.cs
+//cs_include Scripts/Evil/SDKA/CoreSDKA.cs
+//cs_include Scripts/Story/BattleUnder.cs
+//cs_include Scripts/Other/Classes/Necromancer.cs
+using Skua.Core.Interfaces;
+using Skua.Core.Options;
+
+public class AninaQuestReward
+{
+    private IScriptInterface Bot => IScriptInterface.Instance;
+    private CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new();
+    public CoreAdvanced Adv = new();
+    public BladeOfAwe BOA = new();
+    public BuyScrolls Scroll = new();
+    private CoreBLOD BLOD = new();
+    private CoreNSOD NSOD = new();
+
+    public string OptionsStorage = "Burning Weapons";
+    public bool DontPreconfigure = true;
+    public List<IOption> Options = new List<IOption>()
+    {
+        CoreBots.Instance.SkipOptions,
+        new Option<WeaponChoice>("WeaponChoice", "Choose Your weapon", "", WeaponChoice.None),
+    };
+
+    public void ScriptMain(IScriptInterface Bot)
+    {
+        Core.SetOptions();
+
+        Weapon();
+
+        Core.SetOptions(false);
+    }
+
+    public void Weapon(WeaponChoice WeaponChoice = WeaponChoice.None)
+    {
+        Core.OneTimeMessage(
+            "Skua Cannot do this",
+            "Bot Cannot Get \"Volcanic Essence\", as it Requires you to kill an ultra, and skua cannot do this.",
+            true,
+            true
+        );
+
+        if (Bot.Config!.Get<WeaponChoice>("WeaponChoice") == WeaponChoice.None)
+            return;
+
+        if (Bot.Config.Get<WeaponChoice>("WeaponChoice") == WeaponChoice.Burning_Sword_of_Doom || Bot.Config.Get<WeaponChoice>("WeaponChoice") == WeaponChoice.All && !Core.CheckInventory("Burning Sword of Doom"))

```

Comment:
why is this here? i told u to add the `|| Bot.Config.Get<WeaponChoice>("WeaponChoice") == WeaponChoice.All` to the 3 if lines for each weapon (check the example script I sent you)

Reply:
besides that merges are fine